src/main/java/org/skriptlang/skript/bukkit/itemcomponents/equippable/elements/CondEquipCompSwappable.java

```diff
@@ -0,0 +1,66 @@
+package org.skriptlang.skript.bukkit.itemcomponents.equippable.elements;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.conditions.base.PropertyCondition;
+import ch.njol.skript.doc.*;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.bukkit.itemcomponents.equippable.EquippableExperiment;
+import org.skriptlang.skript.bukkit.itemcomponents.equippable.EquippableWrapper;
+
+@Name("Equippable Component - Is Swappable")
+@Description("Whether an item can be swapped by right clicking in it your hand. "
+	+ "Note that equippable component elements are experimental making them subject to change and may not work as intended.")
+@Example("""
+	if {_item} is swappable:
+		add "Swappable" to lore of {_item}
+	""")
+@Example("""
+	set {_component} to the equippable component of {_item}
+	if {_component} is not swappable:
+		make {_component} swappable
+	""")
+@RequiredPlugins("Minecraft 1.21.2+")
+@Since("INSERT VERSION")
+public class CondEquipCompSwappable extends PropertyCondition<EquippableWrapper> implements EquippableExperiment {
+
+	static {
+		Skript.registerCondition(CondEquipCompSwappable.class, ConditionType.PROPERTY,
+			"%equippablecomponents% (is|are) [:un]swappable",
+			"%equippablecomponents% (isn't|is not|aren't|are not) [:un]swappable"
```

Comment:
Likewise i think this syntax could be much more clear about what it actually does. Swapping brings to mind offhand swapping/number key actions and I think it could use some more specific language about it being equipped via clicking.